### PR TITLE
use annotation tracking by default

### PR DESCRIPTION
**Pull request by nabuskey**
Annotation tracking method is a must when using Crossplane with ArgoCD. This PR enables it. 

https://argo-cd.readthedocs.io/en/stable/user-guide/resource_tracking/#additional-tracking-methods-via-an-annotation

### CODE REVIEW

**Review by cmoulliard — COMMENTED** [review]
Can you also mention within the README file that idpbuilder's argocd set by default the property `application.resourceTrackingMethod: annotation`, point to the argocd doc and explain why ?

**Comment by nabuskey** [comment]
@cmoulliard Good idea. Reflected it in the README.

**Review by cmoulliard — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -243,6 +243,19 @@ apiVersion: kind.x-k8s.io/v1alpha4
...
+
+The default manifests for the core packages are available [here](pkg/controllers/localbuild/resources). 
+These are generated by scripts. If you want to make changes to them, see below.
+
+#### ArgoCD
+
+ArgoCD manifests are generated using a bash script available [here](./hack/argo-cd/generate-manifests.sh). 
+This script runs kustomize to modify the basic installation manifests provided by ArgoCD. Modifications include:
+
+1. Prevent notificaiton and dex pods from running. This is done to keep the number of pods running low by default.
```
Comment (jessesanford) [inline README.md:256]: /notificaition/notification/

**Review by jessesanford — APPROVED** [review]
Just a spelling typo and LGTM

**Pull request merged**